plio/settings.py

```diff
@@ -282,3 +284,13 @@
     {"name": "org-admin"},
     {"name": "org-view"},
 ]
+
+CHANNEL_LAYERS = {
+    "default": {
+        # Method 1: Via redis lab
+        # Method 2: Via local Redis
+        # Method 3: Via In-memory channel layer
+        # Using this method.
+        "BACKEND": "channels.layers.InMemoryChannelLayer"
```

Comment:
In-memory is only good for local dev: https://channels.readthedocs.io/en/stable/topics/channel_layers.html?highlight=channels.layers.InMemoryChannelLayer#configuration

Need to set up Redis for deployment.